sunpy/net/dataretriever/sources/lyra.py

```diff
@@ -37,7 +36,7 @@ def _get_url_for_date(self, date, **kwargs):
 
         Parameters
         ----------
-        date : Python datetime object
+        date : `astropy.time.Time`, `~datetime.datetime`, `~datetime.date`
```

Comment:
Do we not just want to list astropy.time here?

Reply:
It works with `datetime` and `date` too (because only a `date.strftime` is used in the function). Given it's a private function, I can make it just `Time` too, though. What do you think?

Reply:
If its private, we can just leave it. Is my take on that.